### ISSUE TITLE
"Erase all my data" button. 

### ISSUE THREAD

**Issue by Carreau** [jupyterlite/jupyterlite-sphinx#121]
This one is technically unnecessary of you understand how jupyterLite work, though this is more a question of User-interface than anything else. 

We had a couple of request on the security mailing list of people asking us to "delete [their] data from [our] servers", in particular some quite panicked folks that had uploaded medical information and got their finger slapped. 

User don't make the difference, or don't understand that the data is in their browser, and when they come back and see that files are still there, they may panic.  

I suggest we add a button "Erase all my data", that pop up a dialog with:
 - a short explanation
 - a "Erase all my data" (that clean any local storage or relevant).

I don't think it is pressing, but if this get a more usage we might start to have requests.

**Comment by jtpio** [comment]
Yeah this is also a popular request in JupyterLite itself: https://github.com/jupyterlite/jupyterlite/issues/9

Two possible ways forward:

- Fix https://github.com/jupyterlite/jupyterlite/issues/9, which should be done anyway at some point
- Provide documentation in `jupyterlite-sphinx` to make it more visible how to use volatile memory storage: https://jupyterlite.readthedocs.io/en/stable/howto/configure/storage.html#volatile-memory-storage
  - or make it the default in `jupyterlite-sphinx`?

**Comment by Carreau** [comment]
I guess once this is in JupyterLite, we can just have a hook to have say a link in the documentation footer when using jupyterlite-sphinx, that allow to erase w/o having to actually launch jupyter-lite. I think that not everyone will think at launching and then erasing.
